test/passing/comment_last.ml

```diff
@@ -1,5 +1,3 @@
 let x = 2
 
-let y = 3
-
-(*comment*)
+let y = 3 (*comment*)
```

Comment:
This is a regression

Reply:
Only if we want to preserve the empty line (that means relying on the input), otherwise it's a simple structure_item and the comment is attached to it and it fits.

Reply:
I think the comment should be treated as a (not simple) structure item otherwise this is changing the meaning of the comment.